### PR TITLE
Persistent notes

### PR DESCRIPTION
**Pull request by analeamaesterling**

### CODE REVIEW

File: src/main/ui/src/App.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
 import SearchByName from "./components/SearchByName/SearchByName";
 import ParkNotes from "./components/Notes/ParkNotes";
 import { useState } from "react";
+
```
Comment (jaylingelbach) [inline src/main/ui/src/App.js:4]: Lets get rid of this empty line

File: src/main/ui/src/components/Notes/ParkNotes.jsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import Note from './Note';
 import AddNote from './AddNote';
 
+
```
Comment (jaylingelbach) [inline src/main/ui/src/components/Notes/ParkNotes.jsx:4]: this should be removed as well :)

File: src/main/ui/src/components/Notes/ParkNotes.jsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
 export default function ParkNotes({ notes, handleAddNote }) {
     return (
         <div className='notes-list'>
-            {notes.map((note) => (<Note id={note.id} text={note.text} date={note.date}/>
+            {notes.map((note, index) => (<Note key={index} text={note.text} date={note.date}/>
```
Comment (jaylingelbach) [inline src/main/ui/src/components/Notes/ParkNotes.jsx:8]: killin' it!

File: src/main/ui/src/components/Notes/AddNote.jsx [hunk 4]
```diff
@@ -5,18 +5,41 @@ export default function AddNote({handleAddNote}) {
...
+              method: "POST",
+              headers:{"Content-Type":"application/json"},
+              body:JSON.stringify(noteText)
+              }).then(()=>{
+                console.log("New Note Added")
+              })
+
         if(noteText.trim().length > 0){
           handleAddNote(noteText);
           //returns add note text to blank box
```
Comment (jaylingelbach) [inline src/main/ui/src/components/Notes/AddNote.jsx:36]: remove all comments plz

File: src/main/ui/src/components/Notes/AddNote.jsx [hunk 5]
```diff
@@ -5,18 +5,41 @@ export default function AddNote({handleAddNote}) {
...
+//           //returns add note text to blank box
+//           setNoteText('')
+//         }
+//     };
+
+    const handleSaveClick = (event) => {
+         event.preventDefault();
+         const noteInfo={noteText}
+         console.log(noteInfo);
+//            fetch("http://localhost:3000/notes/addNotes", {
```
Comment (jaylingelbach) [inline src/main/ui/src/components/Notes/AddNote.jsx:25]: remove all comments plz

File: src/main/ui/src/components/Notes/AddNote.jsx [hunk 6]
```diff
@@ -5,18 +5,41 @@ export default function AddNote({handleAddNote}) {
...
           handleAddNote(noteText);
           //returns add note text to blank box
           setNoteText('')
         }
     };
 
+
+
     return(
     // provides text box to add new note and button to trigger save note event
```
Comment (jaylingelbach) [inline src/main/ui/src/components/Notes/AddNote.jsx:44]: remove all comments plz

File: src/main/ui/src/App.js (hunk 1 above)
Comment (analeamaesterling) [reply src/main/ui/src/App.js:4]: Line removed

File: src/main/ui/src/components/Notes/ParkNotes.jsx (hunk 2 above)
Comment (analeamaesterling) [reply src/main/ui/src/components/Notes/ParkNotes.jsx:4]: Removed

File: src/main/ui/src/components/Notes/AddNote.jsx (hunk 4 above)
Comment (analeamaesterling) [reply src/main/ui/src/components/Notes/AddNote.jsx:27]: Comments removed

File: src/main/ui/src/components/Notes/AddNote.jsx (hunk 6 above)
Comment (analeamaesterling) [reply src/main/ui/src/components/Notes/AddNote.jsx:35]: comments removed

File: src/main/ui/src/components/Notes/AddNote.jsx (hunk 5 above)
Comment (analeamaesterling) [reply src/main/ui/src/components/Notes/AddNote.jsx]: comments removed

**Pull request closed without merging**